### ISSUE TITLE
project is dead

### ISSUE THREAD

**Issue by ketominer** [arachnys/cabot#675]
docker-compose deployment doesn't work
docker-less deployment doesn't work

this is plain sad, at least if the docker-less deployment would still be available, people could use your project.

**Comment by Foorack** [comment]
We deployed this program today, works flawlessly.
Clone the repo, copy the `conf/production.env.template` to `conf/production.env` and then run `docker-compose up -d`. The service is now accessible at http://127.0.0.1:5000. If you want accessible world wide then change docker-compose.yml but recommended is to but it behind an Nginx server.

If you want any kind of help you need to post **outputs** from the commands you have run and **what** you have tried to fix it. What was it exactly that **wasn't** working?! "doesn't work" is the most unhelpful comment which can ever be made in this situation.

Please watch this video and try again:

[![bild](https://user-images.githubusercontent.com/5008081/70388675-30fec780-19b5-11ea-8b37-a893c071992b.png)](https://www.youtube.com/watch?v=53zkBvL4ZB4)

**Comment by kromit** [comment]
But the OP is right. The project seems to be dead or at least abandoned.

- Most of the code base is more than 3 years old.
- Last commit is more than a year old.
- No roadmap or future plans
- No statement from the dev

So, are there some good non cloud alternatives for this?

**Comment by Foorack** [comment]
@kromit We have since migrated away from cabot. We are now running a Grafana instance connected to a MySQL. We then wrote a quick 20-lines NodeJS program which pings our services and just puts it into MySQL.

Although, just because it is old code doesn't mean it doesn't work. I still run a private instance of cabot which I used to monitor for example when the Valve index came out, so this project still works and can be deployed.

This project is **well documented** and can easily be deployed if OP had just read the manual. I'm against the idea that working code should be updated for the sake of it. Code which hasn't been touched in 3 years yet still work well is good code. For anyone who wants more features I would recommend Grafana but this project is not dead for personal uses.
